### PR TITLE
Add node modules

### PR DESCRIPTION
**Pull request by JuanCaicedo**
@danpaz @robbiethegeek 

I've consistently noticed people getting stuck at this exercise, so I thought to write a blog post about it. Any and all feedback welcome 😃

### CODE REVIEW

**Comment by robbiethegeek** [comment]
I think this is really cool, what about getting someone that is doing this nodeschool workshop to test out your explanation maybe thursday?

File: modules-in-node.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
+# Multiple modules in Node.js
+
+I'm one of the organizers for [NodeSchoolDC](http://nodeschool.io/washingtondc/)
+and at of our meetups I see people get stuck on the exact same problem all the
+time. That problem isis exercise 6, `Make it Modular`, which is first time
```
Comment (danpaz) [inline modules-in-node.md]: typo `isis`

Comment (danpaz) [inline modules-in-node.md]: typo `which is first time`

File: modules-in-node.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+/* my-module.js */
+module.exports = function(a, b) {
+  return 'first: ' + a + ', second: ' + b;
+};
+
+/* main.js */
+var func = require('./my-module');
+
+var result = func(1, 2);
+console.log(result); // first: a, second: b
```
Comment (danpaz) [inline modules-in-node.md]: `// first: 1, second: 2` ?

File: modules-in-node.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+/* main.js */
+var func = require('./my-module');
+
+var result = func(1);
+console.log(result); // first: a, second: inside
+```
+
+However, the file that requires that module doesn't have access to any of those
+variable.
```
Comment (danpaz) [inline modules-in-node.md]: typo `variables`

**Comment by danpaz** [comment]
LGTM. +1 to @robbiethegeek's idea!

**Comment by JuanCaicedo** [comment]
Thanks @danpaz and @robbiethegeek! I asked a couple people to review, but no big feedback. I'm going to fix what Dan found and then publish, I'll update it if anyone suggests anything new

**Comment by JuanCaicedo** [comment]
https://medium.com/@_juancaicedo/modules-in-node-js-96332821991a#.9jiyo15lm

**Pull request merged**